### PR TITLE
Allow to use block_property per block type

### PR DESCRIPTION
**Pull request by gisostallenberg**

### CODE REVIEW

File: src/Resources/views/html5/parts/video.html.twig [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{% set block_property = (block_property is defined) ? block_property : 'youtube_identifier' %}
+
+{%- block video_frame -%}
+    <div class='embed-container'>
+        <iframe
+            {% block video_frame_attributes %}
+            src='{% block video_frame_source %}https://www.youtube.com/embed/{{ block[block_property] }}{% endblock %}'
```
Comment (reyostallenberg) [inline src/Resources/views/html5/parts/video.html.twig]: As this *is* a youtube block I would like to keep the template name as is

Comment (gisostallenberg) [reply src/Resources/views/html5/parts/video.html.twig]: I did this to keep it in line with all other blocks which are wrapped in a twig-block statement with exact the name of the file.
I could add a block in a block as such:
```twig
    {% block video %}
    {% block youtube %}
        {% include '@ConnectHollandSuluBlock/html5/parts/youtube.html.twig' %}
    {% endblock %}
    {% endblock %}
```
to keep it bc and in line with all other blocks. Are you ok with this?

Comment (reyostallenberg) [reply src/Resources/views/html5/parts/video.html.twig]: or include the youtube template in video template based on 'type'

Comment (gisostallenberg) [reply src/Resources/views/html5/parts/video.html.twig]: You mean something like
```twig
    {% block video %}
        {% include '@ConnectHollandSuluBlock/html5/parts/'~block.type~'.html.twig' %}
    {% endblock %}
```

This still isn't in line with all the other block **parts**

Comment (gisostallenberg) [reply src/Resources/views/html5/parts/video.html.twig]: Other option is I just set the block_property in src/Resources/views/html5/youtube.html.twig in the twig block video and leave the poorly set up file as is.

Comment (gisostallenberg) [reply src/Resources/views/html5/parts/video.html.twig]: Went for solution 1

**Pull request merged**